CUWesternBlister/CommunityReviews
# created and added code for the correct product table

This pull request drops the old products table on deactivation and then adds the updated product table with the correct product/category associations and all of the most recent products from Luke.

Testing:
1. Make sure you have the code for the plugin pulled from this branch
2. Check your database and see that there the old product table is there. You can do this by querying for a row in the wp_bcr_prodcuts table where categoryID=3. The old table should return no rows. 
3. Deactivate the plugin on your local wordpress site and make sure there is no products table in your database before re-activating.
4. Activate the plugin and ensure that the product table both exists and also now has 920 rows.
